src/Tizen.Sensor/Tizen.Sensor/EventArgs/SensorAccuracyChangedEventArgs.cs

```diff
@@ -24,18 +24,18 @@ namespace Tizen.Sensor
     /// <since_tizen> 3 </since_tizen>
     public class SensorAccuracyChangedEventArgs : EventArgs
     {
-        internal SensorAccuracyChangedEventArgs(TimeSpan timespan, SensorDataAccuracy accuracy)
+        internal SensorAccuracyChangedEventArgs(ulong timestamp, SensorDataAccuracy accuracy)
         {
-            TimeSpan = timespan;
+            Timestamp = timestamp;
             Accuracy = accuracy;
         }
 
         /// <summary>
         /// Gets the time stamp.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        /// <value> The time span. </value>
-        public TimeSpan TimeSpan { get; private set; }
+        /// <value> Timestamp. </value>
+        public ulong Timestamp { get; private set; }
```

Comment:
You are changing the public API. Please read about the TizenFX deprecation policy.

Reply:
The part you mentioned was modified. Subtracting the API will be done in another commit.